src/CosmosDB/CosmosDB/help/Get-AzManagedCassandraClusterNodeStatus.md

```diff
@@ -0,0 +1,91 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.CosmosDB.dll-Help.xml
+Module Name: Az.CosmosDB
+online version: https://docs.microsoft.com/en-us/powershell/module/az.cosmosdb/get-azmanagedcassandraclusternodestatus
+schema: 2.0.0
+---
+
+# Get-AzManagedCassandraClusterNodeStatus
+
+## SYNOPSIS
+Get the node status of ManagedCassandra Cluster.
+
+## SYNTAX
+
+```
+Get-AzManagedCassandraClusterNodeStatus -ResourceGroupName <String> -ClusterName <String>
```

Comment:
Please refer to the design https://github.com/Azure/azure-powershell-cmdlet-review-pr/issues/949